src/outputs/parthenon_hdf5.cpp

```diff
@@ -103,8 +104,10 @@ struct VarInfo {
 
   VarInfo(const std::string &label, const std::vector<std::string> &component_labels,
           int vlen, bool is_sparse, bool is_vector)
-      : label(label), component_labels(component_labels), vlen(vlen),
-        is_sparse(is_sparse), is_vector(is_vector) {
+      : label(label),
+        component_labels(component_labels.size() > 0 ? component_labels
+                                                     : std::vector<std::string>{label}),
```

Comment:
Please use a different signature for component_labels the argument vs the member.

Reply:
The same for the other arguments.

Reply:
Yes - I renamed the members in `VarInfo` to have underscores